src/frontend/app.ts

```diff
@@ -172,7 +173,15 @@ export default (octokitApp: App): Express => {
 	app.get("/jira/atlassian-connect.json", getJiraConnect);
 
 	// Maintenance mode view
-	app.use((req, res, next) => (isMaintenanceMode() ? getMaintenance(req, res) : next()));
+	app.use(async (req, res, next) => {
+		if (!req.session.jiraHost) {
+			// we only activate the maintenance mode for requests that are bound to a jira host
+			next();
+		} else {
```

Comment:
This doesn't mate sense.  There's other calls that might not use or have access to the session.  And you'd think if the maintenance mode is global that it would display the maintenance page.

Reply:
Since the feature flags now support an empty jiraHost, I removed the special case.